source/features/clean-conversation-sidebar.tsx

```diff
@@ -106,7 +106,11 @@ async function cleanSidebar(): Promise<void> {
 	}
 
 	// Development (linked issues/PRs)
-	select('[aria-label="Link issues"] p')?.remove(); // "Successfully merging a pull request may close this issue." This may not exist if issues are disabled
+	const developmentHint = select('[aria-label="Link issues"] p');
+	if (developmentHint) { // This may not exist if issues are disabled
+		removeTextNodeContaining(developmentHint, /^(No branches or pull requests|Successfully merging (a|this) pull request may close (this issue|these issues).)$/);
```

Comment:
I'd rather make them slightly more flexible so it works even if they tweak the text
```suggestion
		removeTextNodeContaining(developmentHint, /No branches or pull requests|Successfully merging/);
```